miro_preprocessor/image_sorter/src/sorter_logic.py

```diff
@@ -85,7 +85,7 @@ def image_tech_captured_mode_is_videodisc(self):
 
     @property
     def is_innopac_id_8_digits(self):
-        return self._search(r"[0-9]{7}[0-9xX]{1}", "image_innopac_id") is not None
+        return self._get("image_innopac_id") is not None and self._search(r"[0-9]{7}[0-9xX]{1}", "image_innopac_id") is not None
```

Comment:
Would this logic be better folded inside `_search`? If you search on an empty key, you automatically get a None result?

Reply:
👍